locale/zh/messages.json

```diff
@@ -4292,11 +4292,11 @@
       ]
     ]
   },
-  "Filter by Service": {
+  "Filter by Framework": {
```

Comment:
If these were changed manually they will need to be reverted and updated using `npm run lingui-extract-with-plugins`. In this case, the entire "Filter by Service" translation should be removed and a new entry with "Filter by Framework" added with an empty translation, so that there is no confusion with the old translation.